### PR TITLE
Thomas/sidebar

### PR DESCRIPTION
**Pull request by thomasgonda3**

### CODE REVIEW

File: src/client/app.tsx [hunk 1]
```diff
@@ -1,9 +1,83 @@
...
+      data: []
+    }
+  }
+
+  componentDidMount(): void {
+    Object.assign(document.body.style, body)
+  }
+
+  getData = async (
+    start: number,
```
Comment (thomasgonda3) [inline src/client/app.tsx]: pass in object instead of all params

File: src/client/components/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    const deviceType = (
+      <div style={largeContainer}>
+        <div style={inputLabel}>Device Type</div>
+        <select
+          style={deviceTypeInput}
+          onChange={e => this.handleDeviceTypeChange(e)}
+        >
+          <option value="">--</option>
+          <option value="Android">Android</option>
+          <option value="IOS">IOS</option>
```
Comment (thomasgonda3) [inline src/client/components/Sidebar.tsx]: create select button component props: size, onChange, options[]

File: src/client/components/Sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+        </select>
+      </div>
+    )
+
+    const deviceInfo = (
```
Comment (thomasgonda3) [inline src/client/components/Sidebar.tsx]: create textarea component with props:
interface InputProps {
  defaultLines: number
  size: 'small' | 'large'
  onChange: Function
}

File: src/client/components/Sidebar.tsx [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+      start: new Date(),
+      end: new Date(),
+      deviceType: '',
+      deviceInfo: '',
+      userMessage: '',
+      userName: ''
+    }
+  }
+
+  handleStartChange(start: Date): void {
```
Comment (thomasgonda3) [inline src/client/components/Sidebar.tsx]: try and make onChange 1 function with param {key: value}
this.setState(param)

File: src/client/app.css [hunk 5]
```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand&display=swap');
```
Comment (thomasgonda3) [inline src/client/app.css:1]: make imported components current with reports-server

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**